protocol/contracts/libraries/Well/LibWell.sol

```diff
@@ -67,6 +67,15 @@ library LibWell {
         revert("Bean not in Well.");
     }
 
+    function getNonBeanIndex(IERC20[] memory tokens) internal pure returns (uint nonBeanIndex) {
```

Comment:
Could we add a comment saying it returns the first ERC20 that's not bean?

Reply:
Added: https://github.com/BeanstalkFarms/Beanstalk/commit/15baa200243eeed13e5b48960e92126fb905ccb6